main.cpp

```diff
@@ -90,6 +96,39 @@ void uthread_yield();
 #endif
 
 /**
+ * Blocks SIGVTALRM interrupts
+ */
+void disable_interrupts() {
+
+    // Block interrupts
```

Comment:
I would calls this disabling signals rather than interrupts, even though he used that language in the lab doc. Interrupts are a kernel-level feature.

Reply:
You're right. It may be more appropriate to say "disable_upcalls", but I called it interrupts because it is often called a timer interrupt. Do you think I should change it to be less ambiguous?

Reply:
I don't think it matters that much, I guess.